frontend/device/pocketbook/device.lua

```diff
@@ -17,6 +17,29 @@ _G.POCKETBOOK_FFI = true
 local function yes() return true end
 local function no() return false end
 
+local browser_app ="/ebrmain/bin/webbrowser.sh"
+
+local function isUrl(s)
+    return type(s) == "string" and s:match("*?://")
+end
+
+local function isCommand(s)
+    return os.execute("which "..s.." >/dev/null 2>&1") == 0
+end
+
+local function runCommand(command)
+      return os.execute(command) == 0
+end
+
+local function getLinkOpener()
+    if jit.os == "Linux" and isCommand(browser_app) then
+        return true, browser_app
+    elseif jit.os == "OSX" and isCommand("open") then
+        return true, "open"
+    end
+    return false
+end
```

Comment:
Quick glance only, but please remove this function.

Reply:
Hi

How then can I point to Pocketbook link opener script?
That is "/ebrmain/bin/webbrowser.sh"

Could you advise?
Thx